dandischema/utils.py

```diff
@@ -1,11 +1,12 @@
 from __future__ import annotations
 
 import re
-from typing import Any, Iterator, List, Union, cast, get_args, get_origin
+from typing import Any, Iterator, List, Sequence, Union, cast, get_args, get_origin
```

Comment:
This is deprecated since py3.9. The prefer one is  `collections.abc.Sequence`

Reply:
hm, indeed it is as many others https://docs.python.org/3/library/typing.html#typing.Sequence as [List](https://docs.python.org/3/library/typing.html#typing.List) and others... we better eventually uniformly adjust such imports

Reply:
ok, did that for now